pylintrc

```diff
@@ -46,7 +46,9 @@ disable=line-too-long,
         no-member,
         import-error,
         fixme,
-        locally-disabled
+        locally-disabled,
+        len-as-condition, # we don't really honor this one
+        syntax-error # happens when pylint tries to process a build in place .so extension
```

Comment:
I don't see it in the docs either; I think it's actually an internal error in pylint that manifests as a file level error. pylint should not be attempting to process .so files (it's not surprising that if you try to interpret them as .py, you get a syntax error...) but that's the behavior I'm seeing.